### PR TITLE
add extra filingLabel prop for sbcfeesummary component

### PR DESCRIPTION
**Pull request by meganawong**
*Issue #:*
https://github.com/bcgov/entity/issues/11660

*Description of changes:*
Added extra filing label for GP/SP in fee summary component.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Review by seeker25 — APPROVED** [review]
Looks good to me.

File: vue/sbc-common-components/src/components/SbcFeeSummary.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@
         <li class="container fee-list__item"
           :key="lineItem.filingType"
           >
-          <div class="fee-list__item-name">{{lineItem.filingType}}</div>
+          <div class="fee-list__item-name">{{filingLabel ? filingLabel : lineItem.filingType}}</div>
```
Comment (severinbeauvais) [inline vue/sbc-common-components/src/components/SbcFeeSummary.vue:19]: FYI, the other way to do this is `filingLabel || lineItem.filingType`.

But this is fine.

File: vue/sbc-common-components/src/components/SbcFeeSummary.vue [hunk 2]
```diff
@@ -71,6 +71,9 @@ export default class SbcFeeSummary extends Vue {
   @Prop({ default: '' })
   private payURL!: string
 
+  @Prop()
+  private filingLabel!: string
```
Comment (severinbeauvais) [inline vue/sbc-common-components/src/components/SbcFeeSummary.vue:75]: FYI, it's usually a good idea to provide a default value (as on line 71) so it's never undefined.

If you're concerned about being able to identify an empty string, all of these are "falsy":
- filingLabel is undefined
- filingLabel is null
- filingLabel is ''

That's why you can use use a simple expression to check if filingLabel is valid.

No change needed.

File: vue/sbc-common-components/src/components/SbcFeeSummary.vue (hunk 1 above)
Comment (severinbeauvais) [reply vue/sbc-common-components/src/components/SbcFeeSummary.vue:19]: This implementation sort of falls down is there are multiple line items (they will all use the override label).

Probably OK for now but may need to be fixed in the future if there are multiple fees and we need to override a fee label.

**Pull request merged**